pkg/bundlemanager/util.go

```diff
@@ -0,0 +1,105 @@
+package bundlemanager
+
+import (
+	"sort"
+	"strings"
+)
+
+var bundlePrefix = "bundles.validator.forsetisecurity.org/"
+
+// BundleAnnotation
+func BundleAnnotation(key string) bool {
```

Comment:
Should we call this `HasBundleAnnotation`?

Reply:
works for me, done